config/config.exs

```diff
@@ -34,6 +34,7 @@ config :help_desk, HelpDesk.Tracer,
   disabled?: true
 
 config :help_desk, SpandexDatadog.ApiServer,
+  batch_size: 2,
```

Comment:
Because this service has such low traffic, we should only wait for 2 traces before sending to datadog instead of the default 10.

Reply:
This is great info, thanks @btkostner 👍